ql/src/java/org/apache/hadoop/hive/ql/exec/repl/ranger/RangerRestClientImpl.java

```diff
@@ -79,32 +79,29 @@
   public RangerExportPolicyList exportRangerPolicies(String sourceRangerEndpoint,
                                                      String dbName,
                                                      String rangerHiveServiceName,
-                                                     HiveConf hiveConf)throws SemanticException {
+                                                     HiveConf hiveConf)throws Exception {
     LOG.info("Ranger endpoint for cluster " + sourceRangerEndpoint);
     if (StringUtils.isEmpty(rangerHiveServiceName)) {
       throw new SemanticException(ErrorMsg.REPL_INVALID_CONFIG_FOR_SERVICE.format("Ranger Service Name " +
         "cannot be empty", ReplUtils.REPL_RANGER_SERVICE));
     }
+    String finalUrl = getRangerExportUrl(sourceRangerEndpoint, rangerHiveServiceName, dbName);
+    LOG.debug("Url to export policies from source Ranger: {}", finalUrl);
     Retryable retryable = Retryable.builder()
       .withHiveConf(hiveConf).withFailOnException(RuntimeException.class)
-      .withRetryOnException(URISyntaxException.class).build();
+      .withRetryOnException(Exception.class).build();
```

Comment:
RuntimeException is a subclass of Exception. If you retry on Exception, RuntimeException will also get retried